### DISCUSSION TITLE
add battery indicator for Maix Amigo

### DISCUSSION THREAD

**Discussion by gyroballmaster** [selfcustody/krux#219] [Ideas]
Since Amigo device has an internal battery, being able to monitor its levels would be nice

**Comment by odudex** [comment]
Battery indicator is already available on development version: ["develop_rc" branch](https://github.com/selfcustody/krux/tree/develop_rc).
You can test this and a lot of other new features from [this repository](https://github.com/odudex/krux_binaries)
You can also join [Krux Telegram Group](https://t.me/SC_Krux) to ask for help and share your ideas

**Reply by gyroballmaster** [reply]
Awesome thanks. I just flashed that repository and enjoy all the new features, especially the QR maker. I was wondering how to make something like a compact seedqr from seedsigner and you made it happen!

**Reply by odudex** [reply]
You can export Compact Seed QR directly after loading a wallet. Just go to Mnemonic-> Compact SeedQR. Swipe sideways to change modes to help you transcript your QR, swipe up to leave
